specfile/changelog.py

```diff
@@ -34,6 +35,24 @@
     "Dec",
 )
 
+_OPENSUSE_CHANGELOG_SEPARATOR = 67 * "-"
+
+
+@unique
+class ChangelogStyle(Enum):
+    """Style of changelog entries"""
+
+    #: "standard" changelog entries as used by Fedora, RHEL, etc.:
```

Comment:
I would mention that the standard style is what RPM is able to parse (unless the openSUSE style is also parseable):

```suggestion
    #: standard changelog entries parseable by RPM (used in Fedora, RHEL, etc.):
```